scan/tls/alert.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2009 The Go Authors. All rights reserved.
```

Comment:
It would be very helpful to have some documentation here. At the minimum we need a revision of the original code this was copied from and a diff to see which changes are custom.